docs/compiling.md

```diff
@@ -205,8 +205,7 @@ the `fable` command in the sample directory. Just be sure to install the npm dep
 the first time.
 
 ```shell
-npm install
-fable
+npm install -g fable-compiler
```

Comment:
This was assuming Fable was already installed globally (as written at the beginning of the document): `npm install` here is to install the local dependencies of each sample and `fable` to compile the sample with the options from `fableconfig.json` (actually, right now only the second instruction would be necessary as `npm install` is included as a `rebuild` event in `fableconfig.json`).

Could you please remove this before merging the PR? Thanks!